fjord/suggest/providers/trigger/api_views.py

```diff
@@ -0,0 +1,92 @@
+from django.utils.decorators import method_decorator
+
+import rest_framework.views
+import rest_framework.response
+from rest_framework.authentication import SessionAuthentication
+
+from fjord.base.utils import (
+    analyzer_required,
+    cors_enabled,
+)
+from fjord.feedback.models import Response as FeedbackResponse
+from fjord.suggest.providers.trigger.forms import TriggerRuleMatcherForm
+from fjord.suggest.providers.trigger.models import TriggerRuleMatcher
+
+
+class CSRFLessSessionAuthentication(SessionAuthentication):
+    def enforce_csrf(self, request):
+        return
+
+
+class TriggerRuleMatchViewAPI(rest_framework.views.APIView):
+    authentication_classes = (CSRFLessSessionAuthentication,)
+
+    @classmethod
+    def as_view(cls, **initkwargs):
+        # Enable CORS
+        view = super(TriggerRuleMatchViewAPI, cls).as_view(**initkwargs)
+        view = cors_enabled('*', methods=['POST'])(view)
+        return view
```

Comment:
Mmmm... I had this return the value at 28, then later spent time fiddling with `csrf_exempt` and another decorator and switched it to this form which was easier to read. I never switched it back.

I'll fix it.